CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1244,7 +1212,7 @@ if nameAndHealthMenuItem and topbarEnabled and not isTenFootInterface then
 	AddItemInOrder(RightMenubar, nameAndHealthMenuItem, RIGHT_ITEM_ORDER)
 end
 
-local Panel3D = require(GuiRoot.Modules.Panel3D)
+--local Panel3D = require(GuiRoot.Modules.Panel3D)
```

Comment:
Woops, I pulled the latest one from the website and added this because it was erroring out the script. Because that file isn't shipped yet. Will fix.